### DISCUSSION TITLE
Implement screen video capture or DOM recorder

### DISCUSSION THREAD

**Discussion by danielvianna** [DustinBrett/daedalOS#59] [Feature Requests]
DOM recorder:
https://robert.ocallahan.org/2020/11/dom-recording-for-web-application-demos.html
https://github.com/rrweb-io/rrweb
https://github.com/oct16/TimeCat

Goal: todo something like https://scrimba.com/scrim/czvKPPsw 
Why?: Allow training videos to be interactive in any application of the OS

There is an actual implementation for the full VS code named Codio: https://www.youtube.com/watch?v=XC2liN3OhA8&feature=youtu.be

Traditional Screen recorder
https://getkap.co/

**Comment by DustinBrett** [comment]
Thank you for the idea! It is indeed a good one and very possible I think. I will add it to my list and post any updates here.

**Comment by DustinBrett** [comment]
I added this a while back. https://github.com/DustinBrett/daedalOS/commit/9833c34599dffbe553533e5ee4bbfff918337d23
